frontend/public/components/RBAC/bindings.jsx

```diff
@@ -475,21 +475,21 @@ const BaseEditRoleBinding = connect(null, {setActiveNamespace: UIActions.setActi
           <div className="separator"></div>
 
           <Section label="Role Binding">
-            <p className="rbac-edit-binding__input-label">Name:</p>
+            <label htmlFor="test––role-binding-name" className="rbac-edit-binding__input-label">Name</label>
```

Comment:
nit: not really sure why the `htmlFor` and `id` are prefixed with  `test––` string.

Reply:
Agreed.  I was simply following the existing pattern on the other inputs.  Change them all?